test/e2e-go/restAPI/restClient_test.go

```diff
@@ -905,3 +936,119 @@ func TestClientPrioritizesPendingTransactions(t *testing.T) {
 	a.True(len(statusResponse.TruncatedTxns.Transactions) == MaxTxns)
 	a.True(statusResponse.TruncatedTxns.Transactions[0].TxID == txHigh.ID().String())
 }
+
+func TestClientCanGetPendingTransactionInfo(t *testing.T) {
+	a := require.New(fixtures.SynchronizedTest(t))
+	var localFixture fixtures.RestClientFixture
+	localFixture.Setup(t, filepath.Join("nettemplates", "TwoNodes50EachFuture.json"))
+	defer localFixture.Shutdown()
+
+	testClient := localFixture.LibGoalClient
+
+	testClient.WaitForRound(1)
+
+	testClient.SetAPIVersionAffinity(algodclient.APIVersionV2, kmdclient.APIVersionV1)
+
+	wh, err := testClient.GetUnencryptedWalletHandle()
+	a.NoError(err)
+	addresses, err := testClient.ListAddresses(wh)
+	a.NoError(err)
+	_, someAddress := getMaxBalAddr(t, testClient, addresses)
+	if someAddress == "" {
+		t.Error("no addr with funds")
+	}
+	a.NoError(err)
+	addr, err := basics.UnmarshalChecksumAddress(someAddress)
+
+	params, err := testClient.SuggestedParams()
+	a.NoError(err)
+
+	firstRound := basics.Round(params.LastRound + 1)
+	lastRound := basics.Round(params.LastRound + 1000)
+	var gh crypto.Digest
+	copy(gh[:], params.GenesisHash)
+
+	prog := `#pragma version 5
+int 1
+loop: byte "a"
```

Comment:
let's check different values

Reply:
I still want to see logging 32 different values in the test:
```
int 0 // loop variable
loop:
dup // copy
int 97 // 0x61 'a'
+ // 97, 98, 99, ...
itob // 'a', 'b', 'c', ...
log
int 1
+
dup
int 32
<
bnz loop
```

Reply:
see Jason's note about `getbyte` above https://github.com/algorand/go-algorand/pull/2529#discussion_r685341889